src/main/java/TennisGame1.java

```diff
@@ -12,65 +24,69 @@ public TennisGame1(String player1Name, String player2Name) {
     }
 
     public void wonPoint(String playerName) {
-        if (playerName == "player1")
+        if (playerName.equals(PLAYER1))
```

Comment:
The player names are passed in the constructor, so it is wrong to hard-code them